xgcm/test/test_duck_array_ops.py

```diff
@@ -14,4 +14,6 @@ def test_concatenate():
     concat_mixed = concatenate([a, b_dask], axis=0)
     assert isinstance(concat, np.ndarray)
     assert isinstance(concat_dask, dsk.array.Array)
-    assert isinstance(concat_mixed, np.ndarray)
+    # assert isinstance(concat_mixed, np.ndarray)
+    # the resulting mixed array is a dask array, not np. due to changes in numpy?
+    assert isinstance(concat_mixed, dsk.array.Array)
```

Comment:
This seems to be a seperate issue, where numpy now returns a dask array if mixed array are passed? a) Is this expected? b) Should we keep this or force a computation on the dask array?

Reply:
Hmm this only works for the py3.7 environment. Any suggestions how to fix this elegantly?

Reply:
this is likely a numpy __array_function__ protocol thing. What are the numpy versions in your different CI configs?

Reply:
We have a consistent output across versions now by enforcing `os.environ["NUMPY_EXPERIMENTAL_ARRAY_FUNCTION"] = "1"`. This however changes the behavior of the output (will now spit out dask arrays for mixed input). Is this acceptable?

Reply:
We can't count on that environment variable to be set. So we need a different approach.

Reply:
it seems to work now that we have dropped the old python versions and require numpy >1.16. Should I see if it also works without that numpy version specified?